docs/source/tutorial/dcgan.rst

```diff
@@ -0,0 +1,358 @@
+DCGAN: Generate the images with Deep Convolutional GAN
+******************************************************
+
+.. currentmodule:: chainer
+
+0. Introduction
+================
+
+In this tutorial, we generate images with **generative adversarial network (GAN)**.
```

Comment:
with **generative adversarial network (GAN)** -> with a **generative adversarial network (GAN)**

Reply:
@bkvogel How about unifying the notation to plural? I mean, generative adversarial networks (GAN). Because the original paper by Ian Goodfellow express it as plural.

Reply:
Sure, I think that would be good.

Reply:
Thanks! I fixed as plural.